roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
> Yes it seems fixing #3734 hasn't been a priority so we get multiple invocations of role dependencies.

#3734 is this PR, I believe you mean some Ansible issue?

Perhaps a viable alternative are plugins, just like we use https://github.com/openshift/openshift-ansible/blob/master/callback_plugins/aa_version_requirement.py to verify Ansible's version before anything else runs.